### PR TITLE
Challenge3 hooks

### PR DESCRIPTION
**Pull request by Ferleomos**
image

**please use your local environment KEY for youtube API**

### CODE REVIEW

File: src/__tests__/HeaderContainer.test.js [hunk 1]
```diff
@@ -3,49 +3,53 @@ import { render, screen } from '@testing-library/react';
 import 'jest-styled-components';
 import renderer from 'react-test-renderer';
 import '@testing-library/jest-dom/extend-expect';
-import ThemeSelector from '../components/Header/ThemeSelector.component';
-import HeaderContainer from '../components/Header/HeaderContainer.component';
+// import ThemeSelector from '../components/Header/ThemeSelector.component';
```
Comment (ragrum) [inline src/__tests__/HeaderContainer.test.js]: Do not keep commented out code, that's why we use git. Just delete it. If you need it back then you can go and grab it from the history.

File: src/hooks/useVideoApi.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                }
+                else{
+                    searchResults = await videoApi.getRelatedVideoData(filterData);
+                }
+
+                if(searchResults && searchResults.items){
+                    setLoadedData(searchResults.items);
+                    setLoading(false);
+                }
+                // if(searchResults){
```
Comment (ragrum) [inline src/hooks/useVideoApi.js]: Ditto, about commented out code.

**Comment by ragrum** [comment]
While no errors, there are some warnings when compiling locally, please take a look:

image

File: src/providers/Youtube/Video.provider.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        const params = {
+            q: queryS,
+            key: process.env.REACT_APP_G_API_KEY,
+            part: 'snippet',
+            type: 'video',
+            maxResults: 10
+        }
+        const videoUrl = new URL(VIDEO_MAIN_URL);
+        videoUrl.search = new URLSearchParams(params).toString();
+        //const response = await fetch('https://606b8b12f8678400172e65f9.mockapi.io/videoApi');
```
Comment (ragrum) [inline src/providers/Youtube/Video.provider.jsx]: Do not keep commented out code.

File: src/providers/Youtube/index.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import {default} from '../../components/Container/index';
```
Comment (ragrum) [inline src/providers/Youtube/index.js]: Why are you importing something isn't used?

**Review by ragrum — CHANGES_REQUESTED** [review]
Please, ping me on Slack when comments are addressed.

File: src/__tests__/HeaderContainer.test.js (hunk 1 above)
Comment (Ferleomos) [reply src/__tests__/HeaderContainer.test.js]: done

File: src/hooks/useVideoApi.js (hunk 2 above)
Comment (Ferleomos) [reply src/hooks/useVideoApi.js]: done

File: src/providers/Youtube/Video.provider.jsx (hunk 3 above)
Comment (Ferleomos) [reply src/providers/Youtube/Video.provider.jsx]: done

File: src/providers/Youtube/index.js (hunk 4 above)
Comment (Ferleomos) [reply src/providers/Youtube/index.js]: removed

**Review by ragrum — APPROVED** [review]

**Pull request merged**